core/ui/public/i18n/language.json

```
@@ -1023,5 +1025,29 @@
     "documentation": "Documentation",
     "helpdesk": "Helpdesk",
     "animated_icons_by": "Animated icons by"
+  },
+  "smarthost": {
+    "title":"Smarthost",
+    "enabled":"Smarthost provider",
+    "hostname_label":"Hostname",
+    "hostname_label_tooltip":"Type the fully qualified domain name where to use the SMTP server",
+    "hostname_placeholder":"smtp.domain.com",
+    "username_label":"Username",
+    "username_label_tooltip":"Type the user account or leave blank if the server requires no authentication",
+    "username_placeholder":"foo@domain.com",
+    "password_label":"Password",
+    "port_label":"Port",
+    "port_label_tooltip":"Write the TCP port where to reach the SMTP server",
+    "port_placeholder":"587",
+    "tls":"Enable STARTTLS encryption",
```

```suggestion
    "tls":"STARTTLS encryption",
```
So it's consistent with the label of other toggles (I forgot this on the first review)